pkg/distro/defs/loader.go

```diff
@@ -429,6 +441,43 @@ func ImageConfig(distroNameVer, archName, typeName string, replacements map[stri
 	return imgConfig, nil
 }
 
+// InstallerConfig returns the InstallerConfig for the given imgType
+// Note that on conditions the InstallerConfig is fully replaced, do
+// any merging in YAML
+func InstallerConfig(distroNameVer, archName, typeName string, replacements map[string]string) (*distro.InstallerConfig, error) {
```

Comment:
The commit message for this mentions `ImageConfig` instead of `InstallerConfig`.

Reply:
Thanks, silly me! Fixed (locally)